source/more-info/unsupported/docker_version.markdown

```diff
@@ -8,7 +8,7 @@ description: "More information on why Docker version marks the installation as u
 The version that is needed by the Supervisor, depends on the features it needs
 for it to work properly.
 
-The current minimum supported version of Docker is: `19.03.0`.
+The current minimum supported version of Docker is: `20.10.17`.
```

Comment:
```suggestion
The current minimum supported version of Docker is: `20.10.1`.
```

https://github.com/home-assistant/supervisor/pull/3853/files#diff-c0437b9a12da6f58751ebaca22c65dc3a97d07af8b4434206fce36c2615c7740R38

Reply:
next ADR will be 20.10.17 - so lets people update to what is needed and not update it 1month later again